README.md

```diff
@@ -55,6 +55,7 @@ MOT:
 * **@cpms** = CPMS regression tests
 * **@elasticSearch** = Elastic Search comparison test suite
 * **@extended** = Extended regression tests
+* **@VEinspection = Vehicle Examiner Inspection test suite
```

Comment:
Missing ** at the end of @VEinspectio

Should look like: **@VEinspection** = Vehicle Examiner Inspection test suite